cpp/benchmarks/copying/shift.cu

```diff
@@ -68,6 +68,19 @@ static void BM_shift(benchmark::State& state)
     cuda_event_timer raii(state, true);
     auto output = cudf::shift(input, offset, *fill);
   }
+
+  auto const elems_read = (size - offset);
+  auto const bytes_read = elems_read * sizeof(int);
```

Comment:
I would like to make the same suggestion that I made on this PR: https://github.com/rapidsai/cudf/pull/14170

Please don't hard code the type here in case the type used in the benchmark changes. Thanks!